### DISCUSSION TITLE
[Feature Request]: New container component variation: Left aligned vertical accent bar

### DISCUSSION THREAD

**Discussion by jakuhn** [GovAlta/ui-components#1352] [Feature request or enhancement]
### Is your feature request related to a problem? Please describe

Currently the design system has limited variations on the 'container' component. The ones available are awesome, but I think we are missing an opportunity to expand the variations. In my service (Court of Justice Criminal Scheduling), we have containers that stack into lists, and the current container variations don't provide a solution that visually supports the list in a way that makes the container list easily readable and navigable. Only having the option for a top accent bar creates too much of a break between listed container items. (please see screenshot explorations in the alternatives considered section)

### Describe the solution you'd like

A container with a left side accent bar, with variations for grey and interactive colours. Please see screenshot of preferred design
<img width="1071" alt="Screenshot 2023-08-16 at 11 53 27 AM" src="https://github.com/GovAlta/ui-components/assets/104456834/5d0ca4c5-e792-4062-b9a6-3f4afa2d3a2a">

### Provide evidence this is a needed component

- Accessibility wise vertical line helps to orient the user with what information is grouped for those that are visually impaired.
- Scan-ability of a container list
- Navigation, helps to clearly define each container unit so the user can see that multiple container cards are a part of the list, and easily find the container they're looking for.

### Describe alternatives you've considered

I've done a handful of visual variations to work through this challenge, screenshots attached.
<img width="1089" alt="Screenshot 2023-08-16 at 11 48 58 AM" src="https://github.com/GovAlta/ui-components/assets/104456834/ba5ae80e-0903-4780-8133-1f3feda1b339">

<img width="1095" alt="Screenshot 2023-08-16 at 11 49 32 AM" src="https://github.com/GovAlta/ui-components/assets/104456834/fa492986-0e4e-412e-829b-9a604b837376">
<img width="940" alt="Screenshot 2023-08-16 at 11 49 52 AM" src="https://github.com/GovAlta/ui-components/assets/104456834/08635f7b-b80b-4b41-a637-43
<img width="938" alt="Screenshot 2023-08-16 at 11 50 07 AM" src="https://github.com/GovAlta/ui-components/assets/104456834/2ef6aeb6-ace4-4c76-9d6b-37c1a473183c">
6e49693ebb">

### Do you have anything already created for this that we can use?

yes

### Are you currently using this proposal inside your own service

yes

### Are you able to assist to bring the feature to reality?

no

### Additional context

Let me know if you have any additional questions!

**Comment by twjeffery** [comment]
@markpawson Can you add some context of your service here as well? I see you are doing something similar in Shelter exchange. Interested to hear of any testing with users on this as well!

**Comment by markpawson6** [comment]
Below is a screenshot of our prod Shelter Exchange product using the Test Env. The idea is stolen from our SCSS Intake Mgmt app. The developer on our team had developed these containers with FED Khean Murphy when they worked together on Intake mgmt. When Housing and Homelessness team needed to display a listing of Shelters it was natural to use this since they had already developed it. We only have one color for the accent bar; however I "think" Intake Mgmt containers used red accent bar to highlight applicants deemed emergency vs regular applicants. Our color accent was more to make a bright contrast with the white background.
Click on any card and a drawer opens to show the Shelter details and allows editing of the fields using text input, and dropdowns. Since this project was a 7 week rush no formal usability testing was done. However we have observed internal GOA workers use the app and have not seen nor heard any issues with this screen. 

![image](https://github.com/GovAlta/ui-components/assets/113639558/a7b3a0a1-d831-4549-81e4-04764196d123)
![image](https://github.com/GovAlta/ui-components/assets/113639558/d35bca68-188d-4975-9bd4-bb085ef134ae)
